static/js/contacts.js

```diff
@@ -54,12 +54,21 @@ const contactFormHTML = `
             </div>
             <div class="slack-container">
                 <div>
+                <div style="position: relative;">
                     <label for="slack-channel-id">Channel ID</label>
-                    <input type="text" class="form-control" id="slack-channel-id" required>
+                    <input type="text" class="form-control" id="slack-channel-id" style="position: relative;" required >
+                    <i class="fa fa-info-circle position-absolute info-icon" rel="tooltip" style="display: block; top: 29px;"
+            title="Specify channel, private group, or IM channel (can be an encoded ID or a name)."
+            id="info-slack-channel-id"></i>
+            </div>
+                </div>
+                <div style="position: relative;">
+                <label for="slack-token">Slack API Token</label>
+                <input type="text" class="form-control" id="slack-token" style="position: relative;" required>
+                    <i class="fa fa-info-circle position-absolute info-icon" rel="tooltip" style="display: block; top: 29px;"
+                    title="Provide a Slack API token (starts with “xoxb”)."
```

Comment:
`xoxb` tokens are specifically bot tokens: https://api.slack.com/authentication/token-types
 Are we only allowing/expecting bot tokens? If so the tooltip should mention that

Reply:
Yes, we only support bot tokens.